src/vs/platform/terminal/common/xterm/shellIntegrationAddon.ts

```diff
@@ -334,7 +334,7 @@ export class ShellIntegrationAddon extends Disposable implements IShellIntegrati
 			}
 			case VSCodeOscPt.CommandLine: {
 				let commandLine: string;
-				if (args.length >= 1 || args.length <= 2) {
+				if (args.length && args.length <= 2) {
```

Comment:
@Tyriar not sure if `args.length <=2` is necessary here....

Reply:
I wanted it to be strict and to fail when it's totally unexpected

Reply:
do we expect `args.length === 1`?

Reply:
oh I see - it can be length 1 or 2